### ISSUE TITLE
pypi release

### ISSUE THREAD

**Issue by dpkp** [mochi/vor#3]
Nice module -- do you have any plans to release via pypi?  If not, tagging an official release would be helpful.

**Comment by twonds** [comment]
We should do both.

**Comment by ralphm** [comment]
Yup, I'll look into doing a release during the PyCon sprints here.

**Comment by ralphm** [comment]
I, finally, opened up PR #11 for this.
